File: 1007 素数对猜想 (20 分).c
#include<stdio.h>
#include<math.h>
int main()
{
    int n,i,j;
    int temp=1;
    int count=0;
    scanf("%d",&n);
    for(i=2;i<=n;i++)
    {
        for(j=2;j<=sqrt(i);j++)
        {
            if(i%j==0)
                break;
        }
        if(j>sqrt(i))
        {
            if(i-temp==2)
            	count++;
            temp=i;
        }
    }
    printf("%d\n", count);
}

File: 1007 素数对猜想 (20 分).py
N=input()
temp=1
count=0
for i in range(2,N+1):  
    for j in range(2,i):
        if (i % j) == 0:
            break
    else:
        if i-temp==2:
            count=count+1
        temp=i
print count
